test/e2e/01-install.sh

```diff
@@ -66,7 +73,7 @@ set -e
 kubectl create secret tls -n tekton-pipelines tekton-results-tls --cert="${SSL_CERT_PATH}/tekton-results-cert.pem" --key="${SSL_CERT_PATH}/tekton-results-key.pem" || true
 
 echo "Installing Tekton Results..."
-kubectl kustomize "${ROOT}/test/e2e/kustomize" | ko apply -f -
+kubectl kustomize "${ROOT}/test/e2e/kustomize" | ko apply --sbom=none -f -
```

Comment:
Why did we disable sbom?

Reply:
For local testing, SBOM isn't useful. SBOM generation adds extra time.

Reply:
note that this is also for e2e testing. At times I have tested with quay.io as my image repo, and pushing the SBOM does not always succeed.

Reply:
This was a known issue but it should be fixed in the latest version: https://github.com/ko-build/ko/issues/532. I haven't tried it recently.

Reply:
I may have to try and reproduce the issue. Sounds like an issue with quay.io.

Reply:
Looks like quay.io doesn't like `spdx+json` media types in some circumstances. Commented in https://github.com/ko-build/ko/issues/660#issuecomment-1412739253 - it looks like ko has support for `text/spdx+json` on the long term roadmap?